mne/coreg.py

```diff
@@ -1223,3 +1227,586 @@ def _scale_xfm(subject_to, xfm_fname, mri_name, subject_from, scale,
                 F_mri_ras, 'ras', 'ras'),
             F_ras_mni, 'ras', 'mni_tal')
     _write_fs_xfm(fname_to, T_ras_mni['trans'], kind)
+
+
+class Coregistration(object):
+    """
+    Class for MRI<->MEG coregistration.
+
+    Parameters
+    ----------
+    info : instance of Info
+        The measurement info.
+    subject : str
+        Name of the subject the data are defined for.
+    subjects_dir : path-like
+        Path to MRI subjects directory.
+
+    Notes
+    -----
+    Internal computation quantities ``parameters`` are in units of (for X/Y/Z):
+    - ``parameters[:3]`` are in radians
+    - ``parameters[3:6]`` are in m
+    - ``paramteres[6:9]`` are in scale proportion
+    """
+
+    def __init__(self, info, subject, subjects_dir):
+        self._default_parameters = [0., 0., 0., 0., 0., 0., 1., 1., 1.]
+        self._parameters = list(self._default_parameters)
+        self._last_parameters = list(self._default_parameters)
+
+        self._n_scale_params = (0, 1, 3)
+        self._n_scale_param = self._n_scale_params[0]
+        self._coord_frames = ('mri', 'head')
+        self._coord_frame = self._coord_frames[0]
```

Comment:
I don't remember why we support both coord_frames but maybe we can start by supporting only one eg mri if it's the default? Would make things simpler....

Reply:
@GuillaumeFavelier I think you can just always use the `mri` coordinate frame in this class